.travis/docker-compose-travis.yml

```diff
@@ -12,8 +12,23 @@ services:
       MYSQL_PASSWORD: "secret"
       MYSQL_DATABASE: "edx_notes_api"
   es:
-    image: elasticsearch:1.5.2
+    image: docker.elastic.co/elasticsearch/elasticsearch:7.8.0
     container_name: es
+    environment:
+      - node.name=es
+      - cluster.name=docker-cluster
+      - cluster.initial_master_nodes=es
+      - bootstrap.memory_lock=true
+      - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
+      - http.port=9200
+    ulimits:
+      memlock:
+        soft: -1
+        hard: -1
+    volumes:
+      - elasticsearch_data:/usr/share/elasticsearch/data
+    ports:
+      - "9200:9200"
```

Comment:
Yes, it was necessary for ES 7.8 to work in order to pass tests on the Travis CI environment. My task was to make it work. I suppose edx devops can fix what they think is suspicious. I will be glad of that. 
These are the docker settings for travis and I didn’t worry much, because it is not as responsible as the devstack environment. 
If I remember correctly, I used this just manual to configurehttps://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html